src/drm_base.h

```diff
@@ -100,6 +100,7 @@ typedef struct _IndexHeader
     int                   async_queue_size;
     std::atomic<uint32_t> queue_index;
     uint32_t              writer_index;
+    std::atomic<bool>     is_rc_running;
```

Comment:
Can we change this to std::atomic<int> and use rc_flag so that it can be used in the future for other purpose?